app/subscriber/src/features/home/home-filters/HomeFilters.tsx

```diff
@@ -24,6 +25,7 @@ export interface IHomeFilterProps {
  */
 export const HomeFilters: React.FC<IHomeFilterProps> = ({ fetch }) => {
```

Comment:
It does not appear that you are using this property in this component either.  Please remove.